src/main/kotlin/mate/academy/model/IsoscelesTrapezoid.kt

```diff
@@ -0,0 +1,17 @@
+package mate.academy.model
+
+class IsoscelesTrapezoid(
+    color: Color,
+    private val base1: Double,
+    private val base2: Double,
+    private val height: Double
+) : Figure(color) {
+    override fun area(): Double = 0.5 * (base1 + base2) * height
+
+    override fun draw() {
+        println(
+            "Figure: isosceles trapezoid, area: ${area()} sq. units, " +
```

Comment:
You may use this functionality to avoid concatenation
https://kotlinlang.org/docs/strings.html#multiline-strings